lametro/templates/base.html

```diff
@@ -9,7 +9,7 @@
 
     <link href="https://maxcdn.bootstrapcdn.com/font-awesome/4.4.0/css/font-awesome.min.css" rel="stylesheet">
     <link href='https://fonts.googleapis.com/css?family=Open+Sans:400,700,300italic' rel='stylesheet' type='text/css'>
-    <link rel="stylesheet" type="text/css" href="{% static 'css/bootstrap.journal.min.css' %}" />
+    <link rel="stylesheet" type="text/css" href="{% static 'css/bootswatch.min.css' %}" />
```

Comment:
@xmedr Looks like `bootswatch` is used for theming. What do we use this CSS file for?

Reply:
This `bootswatch.min.css` seems to contain all bootstrap 5 functionality, in addition to theming. I believe I also have a regular `bootstrap.min.css` in the css directory that isn't actually being used, just in case it turns out that bootswatch was actually missing some things. 

So far it's been sufficient on it's own, and I have a reminder for myself to delete that regular bootstrap file once I'm done. I can delete it now if that makes more sense!

Reply:
@xmedr From the [Bootstrap docs](https://getbootstrap.com/docs/5.0/getting-started/contents/#css-files), it looks like you can just use `bootstrap.min.css` to get all the Bootstrap styles. 

Do we use themes? If not, we could just use the plain Bootstrap CSS file.